src/main/resources/ui/controller/job/SparkController.js

```diff
@@ -29,4 +29,14 @@ class SparkController {
         UiListMethods.addListItem("/newJob/job/jobParameters/maps/appArguments", this._model, "");
     }
 
+    onDeleteAdditionalJar(oEv) {
+        console.log("onDeleteAdditionalJar");
+        UiListMethods.deleteListItem("/newJob/job/jobParameters/maps/additionalJars", this._model, oEv)
+    }
+
+    onAddAdditionalJar() {
+        console.log("onAddAdditionalJar");
```

Comment:
Are the `console.log` supposed to stay like this? If yeas, why not add some more info to it?

Reply:
No, removed.